src/NuGet.Server.Core/Infrastructure/IServerPackageRepository.cs

```diff
@@ -22,6 +22,14 @@ public interface IServerPackageRepository
             ClientCompatibility compatibility,
             CancellationToken token);
 
+        Task<IEnumerable<IServerPackage>> SearchAsync(
+            string searchTerm,
+            IEnumerable<string> targetFrameworks,
+            bool allowPrereleaseVersions,
+            bool allowDelistedVersions,
```

Comment:
The term that we use for this in the Gallery is `unlisted` and not `delisted`. Ideally the `NuGet.Server` code should share the same paradigm--unless it's already defined as "delisted" in here.

Reply:
Currently both words are used for this term. I followed your suggestion and used `unlisted`. Should I rename all occurences of `delisted`?